cosmos/x/evm/plugins/precompile/plugin.go

```diff
@@ -119,42 +120,62 @@ func (p *plugin) SetTransientKVGasConfig(transientKVGasConfig storetypes.GasConf
 // Run implements core.PrecompilePlugin.
 func (p *plugin) Run(
 	evm ethprecompile.EVM, pc vm.PrecompileContainer, input []byte,
-	caller common.Address, value *big.Int, suppliedGas uint64, readonly bool,
-) ([]byte, uint64, error) {
-	// use a precompile-specific gas meter for dynamic consumption
-	gm := storetypes.NewInfiniteGasMeter()
-	// consume static gas from RequiredGas
-	gm.ConsumeGas(pc.RequiredGas(input), "RequiredGas")
-
-	// get native Cosmos SDK context from the Polaris StateDB
+	caller common.Address, value *big.Int, suppliedGas uint64, readOnly bool,
+) (ret []byte, gasRemaining uint64, err error) {
```

Comment:
named errors bad project imo, find a way to not have to do this.

Reply:
don't think its _not_ gothonic actually. cosmos-sdk and geth both do this for panic recovery. its really the only way

Reply:
okay re-enable the linter though and just nolint it here for now